### PR TITLE
Refactored `TypesToLoadAttribute` into `TestExtensionTypesAttribute`

### PR DESCRIPTION
**Pull request by Haplois**
Applied changes in https://github.com/microsoft/vstest/pull/2676.

### CODE REVIEW

**Review by cvpoienaru — APPROVED** [review]

File: src/Adapter/MSTest.CoreAdapter/Properties/TestExtensionTypesAttribute.cs [hunk 1]
```diff
@@ -1,22 +1,17 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using System;
-
-using Microsoft.VisualStudio.TestPlatform.MSTest.TestAdapter;
-using Microsoft.VisualStudio.TestPlatform.ObjectModel.Utilities;
-
-[assembly: TypesToLoad(typeof(MSTestDiscoverer), typeof(MSTestExecutor))]
```
Comment (Sanan07) [inline src/Adapter/MSTest.CoreAdapter/Properties/TestExtensionTypesAttribute.cs:9]: Why did you remove the types from class ad added to AssemblyInfo.cs?

**Review by Sanan07 — APPROVED** [review]

File: src/Adapter/MSTest.CoreAdapter/Properties/TestExtensionTypesAttribute.cs (hunk 1 above)
Comment (Haplois) [reply src/Adapter/MSTest.CoreAdapter/Properties/TestExtensionTypesAttribute.cs:9]: It wasn't necessary here, but to match with https://github.com/microsoft/vstest/pull/2676.
